panel/tests/pane/test_holoviews.py

```diff
@@ -743,3 +743,30 @@ def test_holoviews_property_override(document, comm):
 
     assert model.styles["background"] == 'red'
     assert model.children[0].css_classes == ['test_class']
+
+widget = pn.widgets.IntInput(value=1)
+def _plot(value):
+    return hv.Curve(list(range(value)))
+plot = pn.bind(_plot, widget)
+
+@hv_available
+def test_holoviews_layout_objects(document, comm):
```

Comment:
Please review the below test cases. If you agree that the layout should not contain the `widget_layout` when its empty. Then we should try to find fix. If you know where the fix should be applied I would appreciate you tell me or implement the fix. Thanks.